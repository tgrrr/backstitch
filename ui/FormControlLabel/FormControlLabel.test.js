import React from 'react'
import renderer from 'react-test-renderer'
import { FormControlLabel } from '.'
// import { Checkbox } from '../index';
import { Checkbox } from '@material-ui/core';

it('FormControlLabel: default', () => {
    const component = renderer.create(
        <FormControlLabel label='foo' control={
            <input type="checkbox" id="vehicle2" name="vehicle2" value="Car" />
        // <Checkbox />
    } />
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
