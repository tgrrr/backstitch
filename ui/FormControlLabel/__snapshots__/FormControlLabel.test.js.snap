// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormControlLabel: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Formcontrollabel "
    data-testid="Formcontrollabel"
  >
    <label
      className="MuiFormControlLabel-root"
    >
      <input
        id="vehicle2"
        name="vehicle2"
        type="checkbox"
        value="Car"
      />
      <span
        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
      >
        foo
      </span>
    </label>
  </div>
</div>
`;
